### DISCUSSION TITLE
Fixing the value of z

### DISCUSSION THREAD

**Discussion by ghost** [mboquien/cigale#81] [Q&A]
Is it possible to give a range for the values of z? I have been asked to fix the range of redshift within +-1000 km/s. I am unsure how to do that or if I can add that constraint to the configuration file. Is it possible? 
Thank you for the help.

**Comment by mboquien** [comment]
It depends what you are trying to do exactly. What you could do is give various redshifts within that range in the redshifting module and give a negative redshift in the input flux file. This will set CIGALE to a photometric redshift mode. It will only work if you have a single object. If you have multiple objects you will need to add an uncertainty to the redshift too I think.

**Comment by ghost** [comment]
I am sorry I wasn't clear with my question. So basically we are looking into the redshift range of 0.36 to 2.3. So I have given that range in the redshifting module. But also I want a tolerance of +-1000km/s to make sure my values do not deviate much, how do I give that?

**Comment by mboquien** [comment]
You could always have an irregular grid and give several values within 1000 km/s around each redshift. However, it is not clear to me if the redshifts are already known or not.
